pkg/cmd/version/version.go

```diff
@@ -15,16 +15,31 @@ var log = logf.Log.WithName("version")
 
 // NewCmd creates version cmd which prints version and Build details of the executed binary
 func NewCmd() *cobra.Command {
-	return &cobra.Command{
+	versionCmd := &cobra.Command{
 		Use:          "version",
 		Short:        "Prints the version number of ike cli",
 		Long:         "All software has versions. This is Ike's",
 		SilenceUsage: true,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			LogVersion()
+			short, err := cmd.Flags().GetBool("short")
+			if err != nil {
+				return err
+			}
+			if short {
+				logShortVersion()
+			} else {
+				LogVersion()
+			}
 			return nil
 		},
 	}
+
+	versionCmd.Flags().BoolP("short", "s", false, "prints only version without build details")
+	return versionCmd
+}
+
+func logShortVersion() {
```

Comment:
Why is this private but the other public?

Reply:
b/c the other is called from the operator side (`ike serve`), so not only used by the command itself.

Reply:
bob bob bob

Reply:
I know... but not sure anything else would be less bob bob bob